### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by poo0054** [apache/skywalking#11263]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

this is a bug

### What you expected to happen

skywalking monitor linux

### How to reproduce

https://github.com/apache/skywalking/discussions/11259

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
This is just your local setup. We have the showcase to cover quick setup. Nothing about a bug.

https://skywalking.apache.org/docs/skywalking-showcase/next/readme/

**Comment by poo0054** [comment]
No, you go and try. I tried 9.4.0 and 9.5.0. The variables inside are so different. It was actually modified in 9.6. 8.9.1's variable is also completely different from `SW_OTEL_RECEIVER_ENABLED_OTEL_RULES`.


I followed your documentation completely, and now my telnet port 11800 is completely fine

9.4 
receiver-otel:
  selector: ${SW_OTEL_RECEIVER:default}
  default:
    enabledHandlers: ${SW_OTEL_RECEIVER_ENABLED_HANDLERS:"oc,otlp"}
    enabledOtelRules: ${SW_OTEL_RECEIVER_ENABLED_OTEL_RULES:"apisix,k8s/*,istio-controlplane,vm,mysql/*,postgresql/*,oap,aws-eks/*,windows,aws-s3/*,aws-dynamodb/*"}


9.5：

receiver-otel:
  selector: ${SW_OTEL_RECEIVER:default}
  default:
    enabledHandlers: ${SW_OTEL_RECEIVER_ENABLED_HANDLERS:"otlp-metrics,otlp-logs"}
    enabledOtelMetricsRules: ${SW_OTEL_RECEIVER_ENABLED_OTEL_RULES:"apisix,k8s/*,istio-controlplane,vm,mysql/*,postgresql/*,oap,aws-eks/*,windows,aws-s3/*,aws-dynamodb/*,aws-gateway/*,redis/*,elasticsearch/*,rabbitmq/*"}

**Comment by poo0054** [comment]
Can your 9.5 or 9.4 run?

**Comment by wu-sheng** [comment]
The demo is running for this already. We don't have obligation to prove for you.

**Comment by poo0054** [comment]
this is too bad

**Comment by wu-sheng** [comment]
This is nothing about good or bad. This is for reality. SkyWalking community has only volunteers working on the projects. Majorly, we care about our own use cases for personal interest or our employer's.
We could only help on a limited time, clearly it doesn't include debug for you.

**Comment by poo0054** [comment]
github
